admin/bulk-inactive-transfer.vbs

```diff
@@ -72,109 +77,145 @@ Do
-
 	IF previous_worker_number = "X127CCL" OR previous_worker_number = "X1274EC" or previous_worker_number = "X127966" or previous_worker_number = "X127AP7" or previous_worker_number = "X127CSS" or previous_worker_number = "X127EF8" or previous_worker_number = "X127EF9" or previous_worker_number = "X127EH9" or previous_worker_number = "X127EJ1" or previous_worker_number = "X127EM2" or previous_worker_number = "X127EM3" or previous_worker_number = "X127EM4" or previous_worker_number = "X127EN6" or previous_worker_number = "X127EN8" or previous_worker_number = "X127EN9" or previous_worker_number = "X127EP1" or previous_worker_number = "X127EP2" or previous_worker_number = "X127EQ6" or previous_worker_number = "X127EQ7" or previous_worker_number = "X127EW4" or previous_worker_number = "X127EW6" or previous_worker_number = "X127EW7" or previous_worker_number = "X127EW8" or previous_worker_number = "X127EX4" or previous_worker_number = "X127EX5" or previous_worker_number = "X127EZ2" or previous_worker_number = "X127F3E" or previous_worker_number = "X127F3F" or previous_worker_number = "X127F3J" or previous_worker_number = "X127F3K" or previous_worker_number = "X127F3N" or previous_worker_number = "X127F3P" or previous_worker_number = "X127F4A" or previous_worker_number = "X127F4B" or previous_worker_number = "X127FE2" or previous_worker_number = "X127FE3" or previous_worker_number = "X127FE6" or previous_worker_number = "X127FF1" or previous_worker_number = "X127FF2" or previous_worker_number = "X127FF4" or previous_worker_number = "X127FF5" or previous_worker_number = "X127FG1" or previous_worker_number = "X127FG2" or previous_worker_number = "X127FG5" or previous_worker_number = "X127FG6" or previous_worker_number = "X127FG7" or previous_worker_number = "X127FG9" or previous_worker_number = "X127FH3" or previous_worker_number = "X127FI1" or previous_worker_number = "X127FI3" or previous_worker_number = "X127FI6" or previous_worker_number = "X127FJ2" or previous_worker_number = "X127GF5" or previous_worker_number = "X127Q95" or previous_worker_number = "X127Y86" or previous_worker_number = "X127EP8" or previous_worker_number = "X127EN5" THEN
 		transfer_case_action  = FALSE
-		action_completed = FALSE
+		action_completed = "Excluded"
 	ELSE
 		transfer_case_action = True
+		action_completed = "Confirmed"
 	END IF
-	'msgbox "First: " & previous_worker_number & " " & transfer_case_action
-	MAXIS_case_number 	 = objExcel.cells(excel_row, 2).Value          're-establishing the case numbers for functions to use
-    If MAXIS_case_number = "" then exit do
+
+	MAXIS_case_number 	 = objExcel.cells(excel_row_to_restart, 2).Value          're-establishing the case numbers for functions to use
+    IF MAXIS_case_number = "" THEN EXIT DO 'this should end the script'
     MAXIS_case_number	 = trim(MAXIS_case_number)
-	'msgbox previous_worker_number & " / " & transfer_case_action
+	transfer_to_worker = "X127CCL" 'setting the worker to the closed basket'
```

Comment:
Line 133 - I would add this to the beginning of the code vs. inside the do...loop since it doesn't change.